src/DicomMetaDictionary.js

```diff
@@ -402,6 +419,68 @@ DicomMetaDictionary.sopClassNamesByUID = {
     "1.2.840.10008.5.1.4.1.1.77.1.5.4": "OphthalmicTomographyImage"
 };
 
+DicomMetaDictionary.encodingMapping = {
+    "": "iso-8859-1",
+    "iso-ir-6": "iso-8859-1",
+    "iso-ir-13": "shift-jis",
+    "iso-ir-100": "latin1",
+    "iso-ir-101": "iso-8859-2",
+    "iso-ir-109": "iso-8859-3",
+    "iso-ir-110": "iso-8859-4",
+    "iso-ir-126": "iso-ir-126",
+    "iso-ir-127": "iso-ir-127",
+    "iso-ir-138": "iso-ir-138",
+    "iso-ir-144": "iso-ir-144",
+    "iso-ir-148": "iso-ir-148",
+    "iso-ir-166": "tis-620",
+    "iso-2022-ir-6": "iso-8859-1",
+    "iso-2022-ir-13": "shift-jis",
+    "iso-2022-ir-87": "iso-2022-jp",
+    "iso-2022-ir-100": "latin1",
+    "iso-2022-ir-101": "iso-8859-2",
+    "iso-2022-ir-109": "iso-8859-3",
+    "iso-2022-ir-110": "iso-8859-4",
+    "iso-2022-ir-126": "iso-ir-126",
+    "iso-2022-ir-127": "iso-ir-127",
+    "iso-2022-ir-138": "iso-ir-138",
+    "iso-2022-ir-144": "iso-ir-144",
+    "iso-2022-ir-148": "iso-ir-148",
+    "iso-2022-ir-149": "euc-kr",
+    "iso-2022-ir-159": "iso-2022-jp",
+    "iso-2022-ir-166": "tis-620",
+    "iso-2022-ir-58": "iso-ir-58",
+    "iso-ir-192": "utf-8",
+    gb18030: "gb18030",
+    "iso-2022-gbk": "gbk",
+    "iso-2022-58": "gb2312",
+    gbk: "gbk"
+};
+
+DicomMetaDictionary.defaultEncoding = "latin1";
+
+DicomMetaDictionary.encapsulatedSyntaxes = [
```

Comment:
Same for this.  I would suggest having several categories, and use a Set for this many values.  The types:
Encapsulated
Video
Uncompressed